debian/python-eos-data-distribution.install

```diff
@@ -0,0 +1,3 @@
+eos_data_distribution/producers/70-edd-usb-producer.rules usr/lib/udev/rules.d
+eos_data_distribution/producers/edd-usb-producer.service lib/systemd/system
```

Comment:
i thought we talked about splitting the source into many packages. but ok, let's not be blocked by that.